gear-programs/vft-treasury/src/wasm/tests/vft_treasury.rs

```diff
@@ -0,0 +1,56 @@
+use gtest::{Program, System};
+use sails_rs::prelude::*;
+use utils::{VftTreasury, *};
+
+mod utils;
+
+#[test]
+fn test_treasury() {
```

Comment:
I'd add some more tests, for example:
- check that when there's no token present in mapping it'll return error and not transfer tokens to treasury
- check that `withdraw` can be called only be `ethereum-event-cient`
- check that anyone can call `deposit` after token owner have approved transfer (as it's our use-case with `bridging-payment`)